frontend/src/Dashboard/Header/Header.js

```diff
@@ -24,8 +24,11 @@ class Header extends React.Component {
         <div className="Header__main">
           <h1 className="Header__h1">{"CTXfloods Dashboard"}</h1>
           <ul className="Header__tabs">
-            <li className={pathname.endsWith('crossings') ? 'Header__tab--active' : 'Header__tab'}>
-              <Link to="/dashboard/crossings">Crossing Updates</Link>
+            <li className={pathname.endsWith('crossings/list') ? 'Header__tab--active' : 'Header__tab'}>
+              <Link to="/dashboard/crossings/list">Crossing List</Link>
+            </li>
+            <li className={pathname.endsWith('crossings/map') ? 'Header__tab--active' : 'Header__tab'}>
+              <Link to="/dashboard/crossings/map">Crossing Map</Link>
             </li>
             <li className={pathname.endsWith('addcrossing') ? 'Header__tab' : 'Header__tab'}>
```

Comment:
Sorry I didn't see this at first. It looks like this logic will always return `Header__tab`. Never `--active`.